scripts/User_Management/grade_9_mode.py

```diff
@@ -0,0 +1,41 @@
+from scripts._utils import ssh, pi, utils
+
+host_themes = "pi-themes"
+tvs = ["1", "2", "3", "4"]
+
+TV_OFF_CMD = "echo standby 0 | cec-client -s -d 1"
+TV_ON_CMD = "echo on 0 | cec-client -s -d 1"
```

Comment:
import from elsewhere, don't duplicate.

Reply:
im confused

Reply:
These commands already exist in another module, do they not? Don't repeat yourself, especially with hard coded commands like this. We need to update them, we'll update in one place and forget about the other.

Reply:
Instead of copy and pasting this code from the other module, import the variables/constants.

Reply:
they appear in one other command, should I move them to `TV/_utils.py` ?

Reply:
solved as per [8bec1cf](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/60/commits/8bec1cfe789ef129a71e89d962594cdf4165f7fe)

Reply:
Yes please.